src/main/java/org/jenkinsci/test/acceptance/docker/fixtures/KerberosContainer.java

```diff
@@ -66,17 +69,44 @@
         }
     }
 
+    public String getIpAddress(){
+        String address = null;
+        try {
+            address = Docker.cmd(new String[]{"inspect"}).add(getCid()).popen().asText();
+            address = address.split("IPAddress")[2].split(":")[1].split(",",2)[0].trim().replaceAll("\"", "").replaceAll(",","");
```

Comment:
Why not use `DockerContainer#inspect()` and select the value through `JsonNode`?